poly-synth/src/poly-synth.js

```diff
@@ -60,6 +62,22 @@ class PolySynth {
     this.minQ = -50;
     this.maxQ = 50;
 
+    // The bitcrusher defaults non-information reducing parameters.
```

Comment:
Don't remember right now, but why aren't the bitcrusher nodes defaults set this way already?